src/widgets/youtube/player.tsx

```
@@ -3,94 +3,131 @@
+        if (!firstLoad.current && !isPlaying.current) {
+          const playedVideo = await indexedDB["history"].get(videoId);
+          if (playedVideo) {
+            target.seekTo(playedVideo.duration, true);
+            isPlaying.current = true;
+          }
+        }
+        isPlaying.current = true;
+        startTracking();
+        break;
+      case playerState.BUFFERING:
+        const seekedTime = target.getCurrentTime();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)